talent/onboarding/onboarding.md

```diff
@@ -0,0 +1,68 @@
+
+## Onboarding Process
+
+- This document describes the onboarding process for a new source{d} member.
```

Comment:
@rporres suggested to introduce in the Onboarding Process the picture and editing the landing to add the new mate to the [team page](https://sourced.tech/team/). If there is no picture yet, it would be used the placeholders for new-mates.
Do you think this could be a good idea?

Reply:
At least to add it to the team without a picture, that can be taken at a later moment